internal/controllers/network/tests/import/01-assert.yaml

```diff
@@ -15,14 +15,13 @@ status:
       reason: Success
   resource:
     name: import-external
-    adminStateUp: false
     description: Network import-external from "import" test
-    # Skipped until https://github.com/k-orc/openstack-resource-controller/issues/189 is fixed
-    #dnsDomain: "foo.com"
+    adminStateUp: true
+    dnsDomain: ""
     external: true
-    mtu: 1400
```

Comment:
Why was that changed?

Reply:
Because we don't need to set it. We'll be using default values.